crd/cluster.go

```diff
@@ -0,0 +1,35 @@
+package crd
+
+import (
+	apiextensionsclient "k8s.io/apiextensions-apiserver/pkg/client/clientset/clientset"
+	apierrors "k8s.io/apimachinery/pkg/api/errors"
+	"k8s.io/apimachinery/pkg/util/errors"
+
+	"github.com/grtl/mysql-operator/util"
+)
+
+func MySQLClusterCreateCRD(clientset *apiextensionsclient.Clientset) error {
+	crd, err := util.YAMLToCRD("artifacts/mysqlcrd.yaml")
+	if err != nil {
+		return err
+	}
+
+	_, err = clientset.ApiextensionsV1beta1().CustomResourceDefinitions().Create(crd)
```

Comment:
To reduce repetition, might want to first do `crdInterface := clientset.ApiextensionsV1beta1().CustomResourceDefinitions()` and use that here and on line 27.